app/drivers/gpio/Kconfig.max7318

```diff
@@ -0,0 +1,25 @@
+# MAX7318 GPIO configuration options
+
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+DT_COMPAT_ZMK_GPIO_MAX7318 := zmk,gpio-max7318
```

Comment:
This doesn't match the compatible defined in the yaml binding or used in the code. Accident or intentional?

Reply:
Ah, my bad. The MCP23017 driver doesn't have `gpio-` in front, but the 595 does -- which one should I follow here?

Thanks!

Reply:
Follow the mcp format is fine.